src/Microsoft.Azure.WebJobs.Extensions.Dapr/DaprExtensionConfigProvider.cs

```diff
@@ -54,6 +54,14 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException("context");
             }
 
+            if (!EnvironmentExtensions.ShouldRegisterDaprExtension(this.nameResolver))
```

Comment:
![image](https://github.com/Azure/azure-functions-dapr-extension/assets/22132944/aee45e7b-add7-40f6-b921-4ed50d226831)

If false, it won't create and register converters, but it will register the Dapr singletons (which also starts a web server for Dapr). We should ideally not do any of that. Is there a way to not attach anything to builder if the Dapr extension should not be registered?